routemaster/state_machine/tests/test_gates.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from sqlalchemy import select
+
+from routemaster import state_machine
+from routemaster.db import history
+from routemaster.state_machine.gates import process_gate
+from routemaster.state_machine.types import LabelRef
+from routemaster.state_machine.exceptions import DeletedLabel
+
+
+def assert_history(app_config, entries):
```

Comment:
This looks like a duplicate of the helper in `test_actions.py`; can we share this somehow?

Reply:
Done.